projects/ion/src/lib/core/types/status.ts

```diff
@@ -0,0 +1 @@
+export type StatusType = 'success' | 'info' | 'alert' | 'danger';
```

Comment:
```suggestion
export type StatusType = 'success' | 'info' | 'warning' | 'danger';
```
I think `warning` is the correct name (based on icon name). 🤔